swift/extractor/visitors/StmtVisitor.h

```diff
@@ -1,3 +1,7 @@
+//
+// Created by redsun82 on 26.01.22.
+//
+
```

Comment:
I guess we want to remove this before merge?

Reply:
ouch, these are still there. yep, on to it

Reply:
Yeah, we definitely should.